### DISCUSSION TITLE
How do you have a crosshair in addition to a cursor like labelimg?

### DISCUSSION THREAD

**Discussion by KroitAax** [wkentaro/labelme#1076] [Ideas / Feature requests]
How do you have a crosshair in addition to a cursor like labelimg?

**Comment by KroitAax** [comment]
How can I add a crosshair in addition to the cursor like a labelimg?

**Reply by ganleiboy** [reply]
you can try enhancedlabelme: https://pypi.org/project/enhancedlabelme/

**Comment by KroitAax** [comment]
I am a beginner interested in these, I hope you can tell me, thank you.It's like this positioning crosshair.
![2](https://user-images.githubusercontent.com/88638804/142860125-8b3e8374-9e65-47ef-bc85-527e5121a4b0.png)

**Comment by ice-a** [comment]
hello,
How is this crosshair set?

**Comment by monkeycc** [comment]
https://github.com/wkentaro/labelme/discussions/1051
I also need a cross
Because there is no cross, it is difficult to locate, and it is very difficult to use

**Reply by ganleiboy** [reply]
you can try enhancedlabelme: https://pypi.org/project/enhancedlabelme/

**Comment by SongHeX** [comment]
I also need a cross,Is there a solution

**Reply by ganleiboy** [reply]
you can try enhancedlabelme: https://pypi.org/project/enhancedlabelme/

**Comment by wkentaro** [comment]
Check this PR https://github.com/wkentaro/labelme/pull/1214
![image](https://user-images.githubusercontent.com/4310419/201517244-75ef18de-4b52-49a9-b53e-25f09c03933f.png)

**Reply by KroitAax** [reply]
Thank you for adding, but I found some vulnerabilities when using it. The cross hairs indicate that some pictures taken by mobile phones are not so obvious. How should I make the crosshair obvious?Look forward to your reply.
![image](https://user-images.githubusercontent.com/88638804/201811013-fceb533d-ae44-4ed0-b4d8-68b78c025d77.png)

**Reply by wkentaro** [reply]
Labelme 5.1.0 is out.

```
pip install labelme==5.1.0 --no-cache-dir
```

**Reply by KroitAax** [reply]
This does not seem to solve my problem, should be the image of the problem, thank you for your help.The crosshairs are still hard to see when loading such images.

**Reply by ganleiboy** [reply]
你可以下载labelme5.2.0的源码，然后修改绘制crosshair的参数后，编译labelme进行安装。
具体修改如下：
```
cd labelme/widgets/
在canvas.py脚本中有一个paintEvent()函数，修改其中的：
p.setPen(QtGui.QColor(0, 0, 0))
为：
p.setPen(QtGui.QColor(0, 0, 0), 5)  # 第二个参数就是画笔的粗细
```

**Comment by monkeycc** [comment]
@KroitAax 
用官方版本
不过现在pip没有更新

https://github.com/wkentaro/labelme/tree/25fa31944b4342b4f6562a525f96030a17618f27

你可以下载 覆盖一下  就会有很粗的黑线

**Reply by KroitAax** [reply]
还是不行啊，在加载此类图像的时候，十字线基本上看不见，别的图像却又可以，应该是这类图像的问题，opencv读了重写一下图像还是有问题·

**Reply by wkentaro** [reply]
@KroitAax Maybe you can open an issue with the image.
